ql/src/test/results/clientpositive/llap/bucket_groupby.q.out

```diff
@@ -1753,9 +1753,9 @@ STAGE PLANS:
                   filterExpr: (ds = '103') (type: boolean)
                   Statistics: Num rows: 500 Data size: 89000 Basic stats: COMPLETE Column stats: COMPLETE
                   Top N Key Operator
-                    sort order: ++
-                    keys: key (type: string), value (type: string)
-                    null sort order: zz
+                    sort order: +
+                    keys: key (type: string)
+                    null sort order: z
```

Comment:
I think removing one of the key columns from TNK is too aggressive in this case because both GBYs in the mapper and reducer has the same keys.

Reply:
Yeah, I was concerned about being too aggressive?  Just not sure how to keep the proper aggressiveness and keep the code simple.  

Maybe I should file another Jira for this?  Or would you like this resolved in this patch?

Reply:
Ah, I wrote this comment before reading the other one.

I tihnk I'm ok with doing the other way?  Lemme see if it works, thanks!